NEWS.md

```diff
@@ -9,6 +9,10 @@
   server requirements where Snowflake tokens provide proxied authentication
   while API keys identify users to the Connect server itself.
 
+* Fix for an opaque error while creating a manifest. When Python <= 3.10 is in use
+  and a version requirement in a pyproject.toml file is present, instead of a blocking error,
+  a warning is shown when the tomllib package is not present.
```

Comment:
```suggestion
* Fix an opaque error when creating a manifest in a project using Python <= 3.10 with
  a version requirement in a `pyproject.toml` file. A warning is shown rather than an error
  when the tomllib package is not present.
```